origin/blobclient/client.go

```diff
@@ -95,6 +96,17 @@ func (c *HTTPClient) Addr() string {
 	return c.addr
 }
 
+func (c *HTTPClient) CheckReadiness() error {
+	_, err := httputil.Get(
+		fmt.Sprintf("http://%s/readiness", c.addr),
+		httputil.SendTimeout(5*time.Second),
```

Comment:
Is it possible to make this configurable maybe add an additional field in the [HTTPClient](https://github.com/uber/kraken/blob/b035e118f8173eb6513420beadfdc7c2b4677070/origin/blobclient/client.go#L62) ?

Reply:
Do you mean configurable through the .yaml config or with an option?

Reply:
I think that since all endpoints are non-configurable and have hard-coded timeouts, I'd prefer to stay consistent. If we need to configure the timeout eventually, we can reconsider.